err.c

```diff
@@ -26,6 +26,36 @@ void (*err_exit)(int) = exit;
 const char *err_line_ending = "\n";
 int err_flags = 0;
 
+/* Parse the BST_VERBOSITY environment variable to determine the requested
+   verbosity. Higher numbers are associated with more verbosity, with 9
+   being the most verbose and 0 being the least verbose. The default
+   verbosity level is 1. */
+
+static int parsedverbosity()
+{
+    char *pszVerbosity = getenv("BST_VERBOSITY");
+    if (pszVerbosity == NULL) {
+        return 1;
+    }
+    char *endPtr;
+    int iVerbosity = strtol(pszVerbosity, &endPtr, 10);
+    if (*endPtr != '\0') {
+        // Soemthing went wrong during parsing. This isn't
```

Comment:
Honestly I think I'd prefer if this errored out. It's less surprising than the alternative of making a typo in an envvar and not realizing why it isn't working.